pkg/apis/core/types.go

```diff
@@ -3183,6 +3183,10 @@ type VolumeMountStatus struct {
 	// depending on the mount result.
 	// +optional
 	RecursiveReadOnly *RecursiveReadOnlyMode
+	// ImageRef is the digest of the image used for this volume.
+	// It should have a value that's similar to the pod's status.containerStatuses[i].imageID.
+	// If the volume source is not an ImageVolume, this field will be empty.
+	ImageRef *string
```

Comment:
You want this to match the v1 structure -- you should not need to hand-write conversion code.